backend/lab-service/src/main/java/org/mltooling/lab/api/LabAdminApiHandler.java

```diff
@@ -167,7 +168,13 @@ public StatusMessageFormat resetAllWorkspaces() {
    * @param user - id which should be used to identify the container. Will be prefixed with
    *     {WORKSPACE_SERVICE_PREFIX}
    */
-  public SingleValueFormat<LabService> resetWorkspace(String user) {
+  public SingleValueFormat<LabService> resetWorkspace(String user){
```

Comment:
Code formatting seems wrong here. Make sure to follow the [google-java-format](https://github.com/google/google-java-format). There is an intelliJ plugin: https://plugins.jetbrains.com/plugin/8527-google-java-format/

Reply:
Solved in [6838a86](https://github.com/SAP/machine-learning-lab/pull/26/commits/6838a86687c94f497d3e7cfaeb7f654c71c44c61)